hack/ginkgo-e2e.sh

```diff
@@ -102,6 +102,7 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --repo-root="${KUBE_ROOT}" \
   --node-instance-group="${NODE_INSTANCE_GROUP:-}" \
   --prefix="${KUBE_GCE_INSTANCE_PREFIX:-e2e}" \
+  --os-distro="${OS_DISTRIBUTION:-debian}" \
```

Comment:
also, is the canonical environment variable `OS_DISTRIBUTION` or `KUBE_OS_DISTRIBUTION`? I see both littered throughout, though I suspect `KUBE_OS_DISTRIBUTION` is what we want.